data/locales.json

```
@@ -37,7 +37,7 @@
       },
       {
         "page": "partners",
-        "page_title": "Sharesight Partners",
+        "page_title": "Sharesight US Partners",
```

Updated this for the wrong locale…this should be US, not Global.

```suggestion
        "page_title": "Sharesight Partners",
```